hub/package-manager/winget/pinning.md

```diff
@@ -0,0 +1,235 @@
+
+## Usage
+
+```cmd
+winget pin <subcommand> <options>
+```
+## Arguments
+
+The following arguments are available.
+
+| Argument  | Description |
+|--------------|-------------|
+| **-?, --help** |  Gets additional help on this command. |
+
+## Subcommands
+
+The **pin** command supports the following subcommands.
+
+| Subcommand  | Description |
+|--------------|-------------|
+|  **add** |  Add a new pin. |
+|  **remove** | Remove a package pin. |
+|  **list** | List current pins. |
+|  **reset** | Reset pins |
+
+## add
+
+The **add** subcommand adds a new pin. This subcommand requires that you specify the exact package to pin. If there is any ambiguity, you will be prompted to further filter the **add** subcommand to an exact application.
+
+Usage:
+
+```cmd
+winget pin add [[-q] <query>] [<options>]
+```
+
+### Arguments
+
+| Argument      | Description |
+|-------------|-------------|  
+| **-q,--query**  |  The query used to search for an app. |
+| **-?, --help** |  Get additional help on this command. |
+
+### Options
+
+The options allow you to customize adding pins to meet your needs.
+
+| Option  | Description |
+|-------------|-------------|  
+| **--id**    |  Limits the search to the ID of the application.   |  
+| **--name**   |  Limits the search to the name of the application. |  
+| **--moniker**   | Limits the search to the moniker listed for the application. |  
+| **--tag**   | Limits the search to the tag listed for the application. | 
+| **--cmd, --command**   | Limits the search to the command of the application. |   
+| **-v, --version**  |  Enables you to specify an exact version to pin. The wildcard '*' can be used as the last version part. Changes pin behavior to be [`'gating'`](#pin-types). |  
+| **-s, --source**   |  Restricts the search to the source name provided. Must be followed by the source name. |  
+| **-e, --exact**   |   Uses the exact string in the query, including checking for case-sensitivity. It will not use the default behavior of a substring. |  
+| **--force** | Direct run the command and continue with non security related issues. |
+| **--blocking** | Block from upgrading until the pin is removed, preventing override arguments. Changes pin behavior to be [`'blocking'`](#pin-types). |
+| **--installed** | Pin a specific installed version |
+| **--accept-source-agreements** | Used to accept the source license agreement, and avoid the prompt. |
+| **--header** | Optional Windows-Package-Manager REST source HTTP header. |
+| **--wait** | Prompts the user to press any key before exiting |
+| **--logs, --open-log**  | Open the default logs location. |
+| **--verbose-logs** | Used to override the logging setting and create a verbose log. |
```

Comment:
```suggestion
| **--logs, --open-logs**  | Open the default logs location. |
| **--verbose**, **--verbose-logs** | Enables verbose logging for winget. |
```
Missing `s` in --open-logs, --verbose available as a short option and updated description as per the client